src/iOS.Essentials/BasicViewExtensions/UILabelExtensions.cs

```diff
@@ -67,4 +67,16 @@ public static class UILabelExtensions
         textView.LineBreakMode = breakMode;
         return textView;
     }
+    
+    public static UILabel SetTextWithSpacing(this UILabel label, string text, double spacing)
```

Comment:
Not related to the PR.
Please, create another one if the changes are needed.

Reply:
So, now related